src/test/java/org/apache/ibatis/scripting/defaults/DefaultParameterHandlerTest.java

```diff
@@ -17,9 +17,7 @@
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
@yffstart Please switch back to actual imports.  Avoid star imports.

Reply:
Thanks for reply. I have fix it.